js/app.js

```diff
@@ -1,140 +1,157 @@
 'use strict';
 console.log('Hello');
-//Question 1:
+
 
 let username= prompt('what is your name?').toLowerCase();
 alert('Hello ' + username + ', Welcome to my page!');
 let score = 0;
-//Question 2:
 
-let doesAnyoneLikeChoclate = prompt('Do you like Choclate?');
+//Question 1:
 
-if (doesAnyoneLikeChoclate.toLowerCase() === 'yes' || doesAnyoneLikeChoclate.toLowerCase() === 'y') {
- //console.log('Me too.');
- alert('Me too');
- score++;
- 
-} 
-else if(doesAnyoneLikeChoclate.toLowerCase() === 'no'|| doesAnyoneLikeChoclate.toLowerCase() === 'n') {
+function favoriteChocolate(){
+  let doesAnyoneLikeChoclate = prompt('Do you like Choclate?');
+
+  if (doesAnyoneLikeChoclate.toLowerCase() === 'yes' || doesAnyoneLikeChoclate.toLowerCase() === 'y') {
+    //console.log('Me too.');
+    alert('Me too');
+    score++;
+
+  }
+  else if(doesAnyoneLikeChoclate.toLowerCase() === 'no'|| doesAnyoneLikeChoclate.toLowerCase() === 'n') {
   //console.log('Yes, you do, you\'re in denial');
-  alert('Yes, you do, you\'re in denial.');
+    alert('Yes, you do, you\'re in denial.');
 
+  }
+  else {
+    //console.log('You need to answer with \'yes\' or \'no\' ');
+    alert('You need to answer with \'yes\' or \'no\' ');
+  }
 }
-else {
-//console.log('You need to answer with \'yes\' or \'no\' ');
-alert('You need to answer with \'yes\' or \'no\' ');
-}
-
+favoriteChocolate();
```

Comment:
It's a good idea to put all your function calls in one spot. The standard is to have them all at the end of your document. It's very helpful for bigger projects and debugging!